strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java

```diff
@@ -60,27 +52,19 @@ public void init()
     {
         super.init();
 
-        // Create repositories
-        MutableRepository repositoryReleases = mavenRepositoryFactory.createRepository(REPOSITORY_RELEASES);
-        repositoryReleases.setPolicy(RepositoryPolicyEnum.RELEASE.getPolicy());
-
-        createRepository(STORAGE0, repositoryReleases);
-
-        MutableRepository repositorySnapshots = mavenRepositoryFactory.createRepository(REPOSITORY_SNAPSHOTS);
-        repositorySnapshots.setPolicy(RepositoryPolicyEnum.SNAPSHOT.getPolicy());
-
-        createRepository(STORAGE0, repositorySnapshots);
+    }
 
-        // Generate artifacts
-        generateArtifact(getRepositoryBasedir(STORAGE0, REPOSITORY_RELEASES).getAbsolutePath(),
-                         "org.carlspring.strongbox.metadata:strongbox-metadata",
-                         new String[]{ "3.0.1",
-                                       "3.0.2",
-                                       "3.1",
-                                       "3.2" });
+	@ExtendWith({ RepositoryManagementTestExecutionListener.class})
+    @Test
+    public void testRebuildSnapshotMetadata(@TestRepository(repositoryId = R2, layout = MavenArtifactCoordinates.LAYOUT_NAME, policy = RepositoryPolicyEnum.SNAPSHOT) Repository repository)
+            throws Exception
+    {
+		String repositoryId = repository.getId();
+        String metadataPath = "/storages/" + STORAGE0 + "/" + R2 +
```

Comment:
Cosmetic: Could you please follow the project's coding convention, as described in the wiki? Thanks! :)

Reply:
Done.